curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657e928716b77b2277980276.md

```diff
@@ -7,11 +7,11 @@ dashedName: step-1
 
 # --description--
 
-In this project, you are going to learn about list comprehensions in Python by building a program that can take a `camelCase` or `PascalCase` formatted string and convert that to a `snake_case` formatted string. 
+In this project, you are going to learn about list comprehensions in Python by building a program that can take a `camelCase` or `PascalCase` formatted string and convert it into a `snake_case` formatted string. 
 
 List comprehensions in Python are a concise way to construct a list without using loops or the `.append()` method. Apart from being briefer, list comprehensions often run faster.
 
-Start defining a new function named `convert_to_snake_case()` that accepts a string named `pascal_or_camel_cased_string` as input. For now, add a `pass` statement inside the function.
+To begin, define a new function named `convert_to_snake_case()` that takes `pascal_or_camel_cased_string` as input. Within the function body, include a `pass` statement to mark the starting point of your case conversion.
```

Comment:
The `pass` statement is used to ensure the code remains in a valid syntax, and its inclusion in the function is meant to serve as a placeholder, showing where the case conversion implementation will be added

Reply:
It feels confusing, I wouldn't understand what "to mark the starting point of your case conversion" means.
Pass is added to have valid code, so that python doesn't throw an error for an empty block